content/docs/learning-guide/tutorials/custom-tools/shape-example-cpp.md

```diff
@@ -0,0 +1,383 @@
+---
+linkTitle: Create a Custom Shape Tool in C++
+title: Create a Custom Shape Tool in C++ for Open 3D Engine
+description: Learn to create a custom shape tool in C++ to extend Open 3D Engine (O3DE).
+weight: 200
+toc: true
+---
+
```

Comment:
My suggestion here is to add a screenshot of the sample. This will entice and encourage readers to know what is happening on this page.

Reply:
This should be unblocked now